crates/router/src/db.rs

```diff
@@ -93,8 +94,60 @@ pub trait StorageInterface:
     + blocklist::BlocklistInterface
     + blocklist_fingerprint::BlocklistFingerprintInterface
     + scheduler::SchedulerInterface
-    + payout_attempt::PayoutAttemptInterface
-    + payouts::PayoutsInterface
+    + refund::RefundInterface
+    + reverse_lookup::ReverseLookupInterface
+    + cards_info::CardsInfoInterface
+    + merchant_key_store::MerchantKeyStoreInterface
+    + MasterKeyInterface
+    + payment_link::PaymentLinkInterface
+    + RedisConnInterface
+    + RequestIdStore
+    + business_profile::BusinessProfileInterface
+    + OrganizationInterface
+    + routing_algorithm::RoutingAlgorithmInterface
+    + gsm::GsmInterface
+    + user::UserInterface
+    + user_role::UserRoleInterface
+    + authorization::AuthorizationInterface
+    + user::sample_data::BatchSampleDataInterface
+    + health_check::HealthCheckDbInterface
+    + role::RoleInterface
+    + 'static
+{
+    fn get_scheduler_db(&self) -> Box<dyn scheduler::SchedulerInterface>;
+}
+
+#[cfg(feature = "payouts")]
+#[async_trait::async_trait]
+pub trait StorageInterface:
+    Send
+    + Sync
+    + dyn_clone::DynClone
+    + address::AddressInterface
+    + api_keys::ApiKeyInterface
+    + blocklist_lookup::BlocklistLookupInterface
+    + configs::ConfigInterface
+    + capture::CaptureInterface
+    + customers::CustomerInterface
+    + dashboard_metadata::DashboardMetadataInterface
+    + dispute::DisputeInterface
+    + ephemeral_key::EphemeralKeyInterface
+    + events::EventInterface
+    + file::FileMetadataInterface
+    + FraudCheckInterface
+    + locker_mock_up::LockerMockUpInterface
+    + mandate::MandateInterface
+    + merchant_account::MerchantAccountInterface
+    + merchant_connector_account::ConnectorAccessToken
+    + merchant_connector_account::MerchantConnectorAccountInterface
+    + PaymentAttemptInterface
+    + PaymentIntentInterface
+    + payment_method::PaymentMethodInterface
+    + blocklist::BlocklistInterface
+    + blocklist_fingerprint::BlocklistFingerprintInterface
```

Comment:
@jarnura and @SanchithHegde can we verify this change, its being duplicated for only payouts interface

Reply:
I have added an empty `PayoutsInterface` and `PayoutAttemptInterface` for instances where payouts feature is not enabled. Similar to how it's done for `pub trait Connector:`